drivers/metal/metal_objects.mm

```diff
@@ -222,6 +230,360 @@
 	return blit.encoder;
 }
 
+_FORCE_INLINE_ static MTLSize mipmapLevelSizeFromTexture(id<MTLTexture> p_tex, NSUInteger p_level) {
```

Comment:
The following block was moved from the `RenderingDeviceDriverMetal` into here, to be consistent with the other functions.